code/game/objects/items/manuals.dm

```diff
@@ -312,7 +312,8 @@
 	icon_state = "bookSpaceLaw"
 	starting_author = "Nanotrasen"
 	starting_title = "Space Law"
```

Comment:
```suggestion
	starting_title = "Corporate Regulations" //SKYRAT EDIT CHANGE
```

Reply:
Not a relevant var. This PR is only for fixing wiki links.